src/external/lib/hapi-plugins/company-selection.js

```diff
@@ -7,26 +7,30 @@
  * add-licences route.
  *
  * Both of the above redirections will only take place if the user is
- * attempting to access a route that is configured with an access scope.
+ * attempting to access a route that has not opted out by setting
+ * config.plugins.companySelector.ignore to true.
  */
-const { get, isEmpty } = require('lodash');
+const { get } = require('lodash');
 const SELECT_COMPANY_PATH = '/select-company';
 const ADD_LICENCES_PATH = '/add-licences';
 
 const shouldRedirect = request => {
-  const { companyId } = request.defra;
-  const { path } = request;
-  const access = get(request, 'route.settings.auth.access', {});
+  if (request.method.toLowerCase() === 'get' && request.defra) {
+    const { companyId } = request.defra;
+    const { path } = request;
+    const ignoreRoute = get(request, 'route.settings.plugins.companySelector.ignore', false);
```

Comment:
I wonder if we could use scopes for this?
E.g. if a route requires authentication but not a specific scope, does this indicate that it's a route which does not require company selection?